src/utils.py

```diff
@@ -0,0 +1,43 @@
+import json
+import os
+from typing import Any
+from src.main import Category, Product
+
+
+def read_json(path: str) -> list[dict] | Any:
```

Comment:
В данном случае всегда возвращается список продуктов
```suggestion
def read_json(path: str) -> list[dict]:
```
Для уверенности можно проверить, что `isinstance(data[0], dict)`